packages/director/src/screenshots/s3/config.ts

```diff
@@ -5,7 +5,12 @@ export const S3_FORCE_PATH_STYLE = process.env.S3_FORCE_PATH_STYLE || false;
 export const S3_READ_URL_PREFIX = process.env.S3_READ_URL_PREFIX || undefined;
 export const S3_IMAGE_KEY_PREFIX = process.env.S3_IMAGE_KEY_PREFIX || undefined;
 export const S3_VIDEO_KEY_PREFIX = process.env.S3_VIDEO_KEY_PREFIX || undefined;
-export const UPLOAD_EXPIRY_SECONDS = parseInt(
-  process.env.UPLOAD_EXPIRY_SECONDS || '90',
+export const S3_PRESIGN_DOWNLOAD_URLS = process.env.S3_PRESIGN_DOWNLOAD_URLS || false;
```

Comment:
This is the new bool variable that triggers generating signed download URLs.

Reply:
The previous variable `UPLOAD_EXPIRY_SECONDS` was being used in minio driver as well, we need to preserve it